src/js/edu-benefits/1995/components/IntroductionPage.jsx

```diff
@@ -4,37 +4,79 @@ import { focusElement } from '../../../common/utils/helpers';
 import ProgressButton from '../../../common/components/form-elements/ProgressButton';
 
 class IntroductionPage extends React.Component {
-  constructor() {
-    super();
-    this.goForward = this.goForward.bind(this);
-  }
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
-  goForward() {
+  goForward = () => {
     this.props.router.push(this.props.route.pageList[1].path);
   }
+  goBack = () => {
+    // Wasn't able to figure out how to get react-router to go to
+    //  /education/apply-for-education-benefits/ without having it prepend the
+    //  pathname with /education/apply-for-education-benefits/application/
+    // this.props.router.replace('/education/apply-for-education-benefits/');
+    window.location = "/education/apply-for-education-benefits/";
```

Comment:
The router won't work for this, because the page we're going to is outside of our edu-benefits app.

This should work, but If possible, we should use a regular link for this back button. Something like:

```
<a href="/education/apply-for-education-benefits/application/">
  <button className="usa-button-outline">« Back</button>
</a>
```